voorhoede/tudelft-spacefinder
# Get space images from DatoCMS and add them to the data object of a space

# Changes

- Add image to individual spaces in DatoCMS.
- Merge the data from the space in DatoCMS with the data of the space from the CSV file.
- Show the image in the space card in the front-end.

# Associated issue

It doesn't resolve a specific issue, as this is step one of the implementation of the new design, but it references https://trello.com/c/81Dtcfon/90-update-location-card-according-to-design.

# How to test

1. Open preview link.
2. Navigate to the building "Aula" and click on the space "HAL".
3. An image of a cat should appear in the space card.

# Checklist
